packages/x-components/src/x-modules/scroll/components/window-scroll.vue

```diff
@@ -148,9 +148,9 @@ this state changes, it emits the appropiate X Event to the rest of the applicati
 </script>
 ```
 
-## Customized usage
+### Customized usage
```

Comment:
Is this one needed? I feel like it could be removed and just have the first `Custom usage` header.